engine/lite/components/base/src/main/scala/pl/touk/nussknacker/engine/lite/components/ForEachTransformer.scala

```diff
@@ -11,22 +11,22 @@ import pl.touk.nussknacker.engine.lite.api.utils.transformers.SingleElementCompo
 import scala.collection.JavaConverters._
 import scala.language.higherKinds
 
-object ProcessSplitter extends CustomStreamTransformer {
+object ForEachTransformer extends CustomStreamTransformer {
 
   @MethodToInvoke(returnType = classOf[Object])
-  def invoke(@ParamName("parts") parts: LazyParameter[java.util.Collection[Any]],
+  def invoke(@ParamName("list") list: LazyParameter[java.util.Collection[Any]],
```

Comment:
I wonder if `elements` would be more meaningful?

Reply:
makes sense, I'll change it to `elements`